### PR TITLE
Standardize sample projects, Part 5

### PR DESCRIPTION
**Pull request by nxtn**
Migrates .NET FX projects to SDK-style ones.

Snippets 5000 will crash as long as there is a removed directory.

### CODE REVIEW

**Comment by IEvangelist** [comment]
Thank you for sharing this @NextTurn

Hey @adegeo - wanted you to be aware of this...

> Snippets 5000 will crash as long as there is a removed directory.

**Comment by BillWagner** [comment]
> Snippets 5000 will crash as long as there is a removed directory.

That's my bug. I'll work on a fix today.

**Comment by BillWagner** [comment]
We think we have the CI build fixed for this condition. closing and reopening to test.

File: samples/snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/CustomCopyToDataTableCS.csproj
Comment (BillWagner) [inline samples/snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/CustomCopyToDataTableCS.csproj:6]: I'd prefer not setting C# 8 as the language version for a .NET Framework project. It isn't supported. Was there something in the code that required this?

**Review by BillWagner — COMMENTED** [review]
I had one question here @NextTurn 

Once we resolve it, this is ready to merge.

**Comment by adegeo** [comment]
@NextTurn I deleted the SLN files. They were out of date and referenced VB projects that didn't exist.

File: samples/snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/CustomCopyToDataTableCS.csproj
Comment (NextTurn) [reply samples/snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/CustomCopyToDataTableCS.csproj:6]: This sample is using the "coalescing assignment" language feature.

https://github.com/dotnet/docs/blob/f8887e0d68a67c07d91bcdc4845be70678bd815b/samples/snippets/csharp/VS_Snippets_ADO.NET/DP%20Custom%20CopyToDataTable%20Examples/CS/Program.cs#L320

**Comment by NextTurn** [comment]
CI is failing due to removed *.sln* files, as in Part 2.

**Comment by adegeo** [comment]
@NextTurn We're fine with that. We have a TODO item to exclude removed items from triggering CI. According to the build though, only those two files are causing the problem, so we're considering this a clean build.

I believe we have tomorrow off and I would wait on @BillWagner to merge since he was involved on this originally and left some suggestions for changes. 😄 

Cheers! Thanks for the help!

**Review by BillWagner — APPROVED** [review]
Thanks for helping us update our samples.

I'll approve this now and :shipit:

**Pull request merged**
